### DISCUSSION TITLE
How do I change the position of parameters on visualization screen?

### DISCUSSION THREAD

**Discussion by ghost** [mesa/mesa#1708] [Q&A]
I have many parameters on the visualization screen which have lined up to the left side of the screen as shown below.

<img width="1463" alt="Screenshot 2023-06-01 at 11 13 17 AM" src="https://github.com/projectmesa/mesa/assets/131771458/d436fa17-0fae-4049-b9ce-b6deef82da5a">

is there a way to rearrange some of the parameters and make them appear on the right side where there is some space?

**Comment by rht** [comment]
There is no support for that at the moment. But once we switch to a full Python frontend, it'd be easier to do so. It's currently in the work.

**Reply by ghost** [reply]
Thank you so much for letting me know...
